gpMgmt/bin/gppylib/logfilter.py

```diff
@@ -57,6 +57,14 @@
 # A timezone specifier may follow the timestamp, but we ignore that.
 
 
+# This pattern matches the date and time stamp at the log file name of a
+# GPDB log file. The timestamp format is: YYYY-MM-DD HH:MM:SS or the
+# YYYY-MM-DD (to preserve an old behaviour).
+logNameTSPattern = re.compile(
+    '^.*gpdb-(?P<datetime>\d+-\d+-\d+(?P<time>_\d+)?)\.csv$'
```

Comment:
```suggestion
    '^.*gpdb-(?P<datetime>\d\d\d\d-\d\d-\d\d(?P<time>_\d\d\d\d\d\d)?)\.csv$'
```

Reply:
I suppose the unified pattern is more convenient. Other invalid patterns strptime would recognize